### DISCUSSION TITLE
Users to export other users timesheets

### DISCUSSION THREAD

**Discussion by Jurgens92** [kimai/kimai#2169] [Questions & Answers] [answered]
Hi, on update 1.11.1
I need to users with only the "user" role assigned to be able to export other users time sheets

here is what is configured currently:

![image](https://user-images.githubusercontent.com/40264293/101456031-1fd7f600-393c-11eb-8433-0390ccae7b79.png)

![image](https://user-images.githubusercontent.com/40264293/101455982-0a62cc00-393c-11eb-8dbb-544633cb8bdb.png)

![image](https://user-images.githubusercontent.com/40264293/101455857-dbe4f100-393b-11eb-8a0e-44dcfa0a1504.png)

am I doing something wrong here with the permissions?

thanks

**Comment by kevinpapst** [comment, marked as answer]
> am I doing something wrong here with the permissions?

Next time when you ask a question, please include your problem as well 😄 

Simple answer: you shouldn't. It is not easy to configure and leads to massive data privacy issues. 

If you insist on doing that:
- If you use teams, then create a new team for every user that should be able to perform such an export, make him the teamlead and add all users whose timesheets should be visible as member of the new team.
- **Be cautious** the other solution is to assign the "view_all_data" permission to the `User` role (but that literally deactivates all team permissions)

**Reply by Jurgens92** [reply]
sorted, thank you
